src/components/select/simple-select/simple-select.spec.tsx

```diff
@@ -366,10 +366,10 @@ describe("SimpleSelect", () => {
         });
 
         describe("and the focus triggered by mouseDown", () => {
-          it("then that prop should not be called", () => {
+          it("then that prop should not be called more than once", () => {
             simulateSelectTextboxEvent(wrapper, "mousedown");
             simulateSelectTextboxEvent(wrapper, "focus");
-            expect(onOpenFn).not.toHaveBeenCalled();
+            expect(onOpenFn).toHaveBeenCalledTimes(1);
```

Comment:
**question:** The test description refers to `... not be called` but the assertion has been changed to `toHaveBeenCalled`. Is this correct?

Reply:
Yeah, see the response to James: https://github.com/Sage/carbon/pull/6989/files#r1787414429